skema/program_analysis/CAST/matlab/tests/test_single_assignment.py

```diff
@@ -0,0 +1,39 @@
+from skema.program_analysis.CAST.matlab.matlab_to_cast import MatlabToCast
+from pathlib import Path
+
+# Test the CAST returned by processing a single line MATLAB source file
+
+# The source code for the single line assignment test is 'x = 5'
+# The correct CAST for this single line assignment would be a single node: 
+#   node_type 'Assignment' 
+#   left.val.name 'x'
+#   right.value '5'
+#   right.value_type 'Integer'
+
+def test_single_assignment():
+    """ Tests whether the parser can handle a single assignment statement """
+    filepath = Path(__file__).parent / "data/single_line_assignment.m"
```

Comment:
Let's change `filepath = Path(__file__).parent / "data/single_line_assignment.m"` to `filepath = Path(__file__).parent / "data" / "single_line_assignment.m"`

Reply:
Good idea.  Done.